terraform/aks/application.tf

```diff
@@ -38,5 +36,4 @@ module "web_application" {
   docker_image = var.docker_image
   command      = ["/app/docker-entrypoint.sh", "-m", "-f"]
   probe_path   = null
-
 }
```

Comment:
SE also has a sidekiq app in paas. Number of instances is 0 for review, but this will need to be added at some point.